app/assets/stylesheets/mobile/discourse.scss

```diff
@@ -9,16 +9,14 @@ body {
   background-color: var(--secondary);
 }
 
-.ios-device {
-  textarea {
-    background-color: var(--secondary);
-    font-size: var(--font-size-ios-input);
-    -webkit-tap-highlight-color: transparent;
-  }
+textarea {
+  background-color: var(--secondary); // TODO: why was this iOS-only
+  font-size: var(--font-size-ios-input);
+  -webkit-tap-highlight-color: transparent;
```

Comment:
Per MDN:

> -webkit-tap-highlight-color is a non-standard CSS property that sets the color of the highlight that appears over a link while it's being tapped.

so… does it even do anything for `textarea` elements? 🤔 or input elements like the style below?

Reply:
Weird! I'll remove it